lib/acm_bot/bot.ex

```diff
@@ -26,6 +26,12 @@ defmodule AcmBot.Bot do
     answer(context, "#dep, siempre saludaba\nTaluega #{user}!")
   end
 
+  # Versión precaria del comando armario. Es muy mejorable con las ideas de @IronJanoWar
+  def handle({:command, "armario", msg}, context) do
```

Comment:
I usually like to:

1) Group the messages that are about the same, in this case, commands with commands, I would place this handle right after the start command

2) Be explicit with the commands, ex_gram have this:

``` elixir
defmodule T do
  use ExGram.Bot, name: ...

  command "start"
  command "armario"

  def handle({:command, :start, msg}, ..)

  def handle({:command, :armario, msg}, ..)
end
```

Reply:
I don't understand what do you want to say with the second part...

Reply:
Well now I get it. But I don't see any important difference using any method.

Reply:
There are no important difference, as I said *I usually like to*

I do in that way because I like to have explicit and defined the commands that my bot will handle, but it's just personal preference 😂

Reply:
It's up to you @Alfedi, we trust on you :D